cx-portal/src/components/overlays/UpdateCompanyRole/index.tsx

```diff
@@ -99,6 +134,57 @@ export default function UpdateCompanyRole({ roles }: { roles: string[] }) {
     }
   }
 
+  const handleCheckedAgreement = (
+    checked: boolean,
+    agreement: AgreementsData
+  ) => {
+    if (checked) {
+      checkedAgreementsIds.indexOf(agreement.agreementId) <= 0 &&
+        setCheckedAgreementsIds([
+          ...checkedAgreementsIds,
+          agreement.agreementId,
+        ])
+    } else {
+      const index = checkedAgreementsIds.indexOf(agreement.agreementId)
+      if (index > -1) {
+        checkedAgreementsIds.splice(index, 1)
+        setCheckedAgreementsIds([...checkedAgreementsIds])
+      }
+    }
+  }
+
+  const handleSubmit = async () => {
+    setLoading(true)
+    const fetchAgreements = (agreements: AgreementsData[]) => {
+      return agreements.map((agreement: AgreementsData) => {
+        return {
+          agreementId: agreement.agreementId,
+          consentStatus: AgreementStatus.ACTIVE,
+        }
+      })
+    }
```

Comment:
I think you can write this simpler without those `return`s:

`     const fetchAgreements = (agreements: AgreementsData[]) =>`
`      agreements.map((agreement: AgreementsData) => ({`
`          agreementId: agreement.agreementId,`
`          consentStatus: AgreementStatus.ACTIVE,`
`      }))`